pkg/lwip/contrib/netdev/lwip_netdev.c

```diff
@@ -308,6 +323,22 @@ static void *_event_loop(void *arg)
             netdev_t *dev = msg.content.ptr;
             dev->driver->isr(dev);
         }
+#if IS_USED(MODULE_LWIP_IPV4)
+        else if (msg.type == LWIP_NETDEV_MSG_TYPE_LINK_UP) {
+            netdev_t *dev = msg.content.ptr;
+            if (dev->driver->get(dev, NETOPT_LINK, NULL, 0) > 0) {
+                if (dev->event_callback) {
+                    dev->event_callback(dev, NETDEV_EVENT_LINK_UP);
+                } else {
+                    DEBUG("No event callback\n");
+                }
+            } else {
```

Comment:
What about `NETDEV_EVENT_LINK_DOWN`? Does `lwip`  even make use such event?

A "proper" way would be keeping a periodic maintenance routine to check the link status.  Currently, you will only see the first link up and never check the state again.